test/index.js

```diff
@@ -107,7 +107,8 @@ describe('Basic behaviors', function() {
     done();
   });
 
-  it('treats 1st argument as an empty object when it is invalid', function(done) {
+  it('treats 1st argument as an empty object when it is invalid',
```

Comment:
Prefer ignoring line length in tests over putting onto a newline.  Gulp has a separate ruleset for tests that overrides line length.